iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/DeviceRegistryExample.java

```diff
@@ -84,11 +84,19 @@
  * <code>
  * $ mvn clean compile assembly:single
  * $ mvn exec:java \
```

Comment:
We've found that some libraries don't like exec:java as the shutdown code isn't called properly.  We are moving to exec:exec.

Reply:
SGTM, will resolve once updated.